x-pack/platform/plugins/shared/maps/public/inspector/vector_tile_adapter/components/requests_view_callout.tsx

```diff
@@ -14,7 +14,7 @@ export function RequestsViewCallout() {
     <EuiCallOut
       size="s"
       title={i18n.translate('xpack.maps.inspector.vectorTile.requestsView', {
-        defaultMessage: `You're viewing vector tile search requests. To view requests submitted to the search API, set View to Requests.`,
+        defaultMessage: `You're viewing vector tile search requests. To view requests submitted to the search API, select Requests tab.`,
       })}
       iconType="iInCircle"
     />
```

Comment:
Under what conditions is this callout shown? The message strikes me as overly obvious, but perhaps I'm not understanding its intended purpose. Do we actually need this callout?

Reply:
Visualizations like lens show "Request" view by default. Maps shows the "Vector tile" view by default. They are different view for different type of requests. This callout is for maps vector tile inspector view.  Map data can be loaded from vector tile requests or search DSL requests. Users do not really know which to expect as it depends on the implementation of the layer.